README.md

```diff
@@ -5,7 +5,9 @@
 <a href="https://github.com/ambv/black"><img alt="Code style: black" src="https://img.shields.io/badge/code%20style-black-000000.svg"></a>
 </p>
 
-## Getting started
+# Getting started
+
+There are two ways to run the tests, using Docker or with a local virtual environment. Not all tests require Selenium, but for the ones that do you will need `chromedriver` locally installed (in MacOS with homebrew run `brew cask install chromedriver`).
```

Comment:
I could be wrong, but I believe we want to enforce the notion of there only being one way to run this. That helps mitigate the issues around documenting setup for all platforms, which turns into a tangled mess of unmaintainable documentation. (see cnx-archive for example 😬)

Reply:
Hm... so then we should delete the part of the README for "preparing the project locally" then?

Reply:
So then should we only make mention of Docker and omit local installation?